torch/_torch_docs.py

```diff
@@ -106,6 +106,15 @@ def merge_dicts(*dicts):
         the pinned memory. Works only for CPU tensors. Default: ``False``.
 """)
 
+tf32_notes = {
```

Comment:
Instead of a large note on each function affected by this @ngimel and I suggest using a style like https://pytorch.org/docs/master/generated/torch.div.html?highlight=div#torch.div where the functions simply state "Supports ..." and one of the items they support is "TensorFloat32" which is a link to https://pytorch.org/docs/master/notes/cuda.html#tensorfloat-32-tf32-on-ampere-devices. Otherwise the documentation will become too cluttered with large notes.

Reply:
There is no existing list that I can add to, so I added a sentence:

![image](https://user-images.githubusercontent.com/1032377/93278157-b8653c80-f778-11ea-9ca5-de939a76e81b.png)